miio/devicestatus.py

```diff
@@ -143,7 +134,8 @@ def sensor(name: str, *, unit: str = "", **kwargs):
     """
 
     def decorator_sensor(func):
-        property_name = func.__name__
+        property_name = str(func.__name__)
+        qualified_name = str(func.__qualname__)
```

Comment:
Should this change also be done for other decorators for the sake of consistency?

Reply:
yes it schould be done for all of them, I think I did.

Reply:
I did it for setting and sensor and those are the only 2 there are now (I think)

Reply:
@rytilahti are there any others?

Reply:
Oh, sorry, I think that's right. Let's get this merged and take care of any downfall in followup PRs :+1: